openid-4-verifiable-presentations-1_0.md

```diff
@@ -742,7 +649,7 @@ Each entry in `credentials` MUST be an object with the following properties:
 the constraints in `credential_sets`. The value MUST be a non-empty string
 consisting of alphanumeric, underscore (`_`) or hyphen (`-`) characters.
 Within the Authorization Request, the same `id` MUST NOT
-be present more than once.
+be present more than once (whether for credentials or for claim sets, defined below).
```

Comment:
Is this change saying that ids must be completely unique within a query so a claim query id cannot use the same value as a credential query id and that the claim query id cannot be reused across credential queries?

e.g. would this be invalid?
```
{
  "credentials: [
    {
      "id": "query_1"
      ...
      "claims": [
        {"id": "query_1", "path": ["last_name"]},
        {"id": "query_2", "path": ["first_name"]},
      ]
    },
    {
      "id": "query_2",
      ...
      "claims": [
        {"id": "query_1", "path": ["last_name"]},
        {"id": "query_2", "path": ["first_name"]},
      ]
    }
  ]
}
```

Reply:
That's what it says, but not what I originally intended. I will revert this change.